hazenlib/ACRObject.py

```diff
@@ -151,7 +150,7 @@ def find_phantom_center(self):
         radius = int(detected_circles[2])
         return centre, radius
 
-    def get_mask_image(self, image, mag_threshold=0.05, open_threshold=500):
+    def get_mask_image(self, image, mag_threshold=0.07, open_threshold=500):
```

Comment:
it would be great to be able to test what this threshold value should be on a range of datasets from various scanners with various magnetic strengths. - not a blocker for merging this PR but good for fine-tuning the measurements